src/static/js/pad.js

```diff
@@ -669,6 +678,12 @@ const pad = {
       $('#options-stickychat').prop('checked', false); // set the checkbox for off
     }
   },
+  determineFocus: (asNowConnectedFeedback) => {
+    const focus = padcookie.getPref('focusOnEditor');
```

Comment:
I don't think this setting should be persisted in a cookie.

Reply:
I agree.

Reply:
Pushed a change that removes the cookie.